### PR TITLE
Tributech DSK Nodes

### PR DESCRIPTION
**Pull request by mayrmax**
Add Nodes for [Tributech DSK](https://docs.tributech.io/docs/introduction/overview) API's:

- Data-API
- Trust-API
- Twin-API
- Catalog-API

I will open a pull request for the documentation soon.

I did my best to follow the node submission checklist, if something is missing please let me know.

### CODE REVIEW

**Comment by krynble** [comment]
Hey @mayrmax 

Thank you for your contribution.

I would like to make some testing before providing feedback. How can I get an account? I went to your website and saw that I'd need to get in touch with a representative and stuff.

Could you share with me a test account? You can contact me directly at omar@n8n.io.

File: packages/nodes-base/nodes/Tributech/descriptions/catalog-api/ManageModelsDescription.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+				description: 'Revoke Model',
+				value: 'revokeModel',
+			},
+		],
+		default: 'addNewModels',
+	},
+] as INodeProperties[];
+
+export const manageModelsFields = [
+	{
```
Comment (krynble) [inline packages/nodes-base/nodes/Tributech/descriptions/catalog-api/ManageModelsDescription.ts:45]: For `Get all`-like operations we always fetch a single page by default, with the maximum number of items in a single page and add a toggle named `Return All` that automatically iterates over all pages and returns all data in a single execution.

Could you please make this change? This makes it consistent with n8n's expected behavior.

File: packages/nodes-base/nodes/Tributech/descriptions/twin-api/RelationshipsDescription.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+			show: {
+				resource: [
+					'relationships',
+				],
+				operation: [
+					'getAllRelationships',
+				],
+			},
+		},
+		options: [
```
Comment (krynble) [inline packages/nodes-base/nodes/Tributech/descriptions/twin-api/RelationshipsDescription.ts:76]: Same as above, related to `Get all` operations. We can remove page number and page size and fetch a single page by default and a `Return all` toggle that makes the node automatically iterate over all pages and return all data in a single execution.

File: packages/nodes-base/nodes/Tributech/descriptions/twin-api/TwinsDescription.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+				resource: [
+					'twins',
+				],
+				operation: [
+					'getAllTwins',
+					'getTwinsByModelId',
+				],
+			},
+		},
+		options: [
```
Comment (krynble) [inline packages/nodes-base/nodes/Tributech/descriptions/twin-api/TwinsDescription.ts:72]: Same here

**Review by krynble — CHANGES_REQUESTED** [review]

**Comment by Joffcom** [comment]
Hey @mayrmax,

Thanks for the contribution, We have now created a Community Node Repository which is our preferred method for new node submissions.

Once a node has been published to the Community Node Repository users will be able to install the node from their n8n interface and will be able to use your node quicker, You will also be able to update the node as needed without needing to wait for the review process.

In the future we may reach out to you about including your node in our main release, But for now I am going to close this PR.

You can find more information in the links below, If you have any questions about this or need any help let us know.

Building Community Nodes: https://docs.n8n.io/integrations/community-nodes/build-community-nodes/
Submitting Community Nodes: https://docs.n8n.io/integrations/creating-nodes/deploy/submit-community-nodes/
Published Nodes: https://www.npmjs.com/search?q=keywords:n8n-community-node-package

**Pull request closed without merging**